_data/simple-icons.json

```
@@ -2985,6 +2985,11 @@
             "hex": "000000",
             "source": "https://unity.com/"
         },
+        {
+            "title": "Unreal Engine",
+            "hex": "2e2e2c",
```

For concistency we prefer to have all hex values using capitals

```suggestion
            "hex": "2E2E2E",
```